connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -117,6 +118,20 @@ public static synchronized void authenticate(Activity activity,
         activity.startActivityForResult(intent, requestCode);
     }
 
+    /**
+     * Get a valid Access Token. If a non-expired one is available, that will be given to the
+     * callback {@code onSuccess(String accessToken)} method.
```

Comment:
Consider mentioning that the token may expire or become revoked before being used. If so, the client should try to refresh the tokens explicitly.